File: MORM.Java/src/MORM/Java/Models/Venda/Regraimposto.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "REGRAIMPOSTO")
public class Regraimposto extends CollectionItem
{
    @CampoAttribute(Campo = "ID_REGRAFISCAL", Tipo = CampoTipo.Key)
    public Integer Id_Regrafiscal;
    @CampoAttribute(Campo = "CD_IMPOSTO", Tipo = CampoTipo.Key)
    public Integer Cd_Imposto;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "PR_ALIQUOTA", Tipo = CampoTipo.Req)
    public Float Pr_Aliquota;
    @CampoAttribute(Campo = "PR_BASECALCULO", Tipo = CampoTipo.Req)
    public Float Pr_Basecalculo;
    @CampoAttribute(Campo = "CD_CST", Tipo = CampoTipo.Req)
    public String Cd_Cst;
    @CampoAttribute(Campo = "CD_CSOSN", Tipo = CampoTipo.Nul)
    public String Cd_Csosn;
    @CampoAttribute(Campo = "IN_ISENTO", Tipo = CampoTipo.Req)
    public String In_Isento;
    @CampoAttribute(Campo = "IN_OUTRO", Tipo = CampoTipo.Req)
    public String In_Outro;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Regraimposto.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("REGRAIMPOSTO")]
    public class Regraimposto : CollectionItem
    {
        [Campo("ID_REGRAFISCAL", CampoTipo.Key)]
        public int Id_Regrafiscal { get; set; }
        [Campo("CD_IMPOSTO", CampoTipo.Key)]
        public int Cd_Imposto { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.Req)]
        public double Pr_Aliquota { get; set; }
        [Campo("PR_BASECALCULO", CampoTipo.Req)]
        public double Pr_Basecalculo { get; set; }
        [Campo("CD_CST", CampoTipo.Req)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.Nul)]
        public string Cd_Csosn { get; set; }
        [Campo("IN_ISENTO", CampoTipo.Req)]
        public string In_Isento { get; set; }
        [Campo("IN_OUTRO", CampoTipo.Req)]
        public string In_Outro { get; set; }
    }
}